app/shared/db/breakpoints.server.ts

```diff
@@ -13,9 +13,18 @@ export const load = async (treeId?: Tree["id"]) => {
     throw new Error("Tree ID required");
   }
 
-  return await prisma.breakpoints.findUnique({
+  const breakpoint = await prisma.breakpoints.findUnique({
     where: { treeId },
   });
+
+  if (breakpoint === null) {
+    throw new Error("Breakpoint not found");
+  }
+
+  return {
+    ...breakpoint,
+    values: JSON.parse(breakpoint?.values),
```

Comment:
since you now added the null check
```suggestion
    values: JSON.parse(breakpoint.values),
```